lib/src/state_manager/rx/rx_core/rx_interface.dart

```diff
@@ -78,7 +81,7 @@ abstract class DisposableInterface {
   /// Or dispose objects that can potentially create some memory leaks,
   /// like TextEditingControllers, AnimationControllers.
   /// Might be useful as well to persist some data on disk.
-  Future<void> onClose() async {}
+  void onClose() async {}
```

Comment:
Won't that produce an error in Get.delete?

Reply:
I'm going to merge it, for backwards compatibility, but I think this will generate a warning in Get.delete

Reply:
I think it fixes the error that we had when generating the override in the IDEs (Future<Function>). Anyway, I tested an async operation on onClose, and code execution was fine